CONTRIBUTING.md

```diff
@@ -46,9 +46,10 @@ Following these guidelines will help us to merge your pull requests smoothly:
 ## Commit Guidelines
 
 The first line describes the change made. It is written in the imperative mood,
-say what happens when the patch is applied. Keep it short and simple. The first
-line should be less than 50 characters, sentence case, and does not end in a
-period. Leave a blank line between the first line and the message body.
+and should say what happens when the patch is applied. Keep it short and
+simple. The first line should be less than 50 characters, sentence case, and
```

Comment:
I'd love to have this discussion, but I think that's an issue, not something for this PR.